source/fem-dem/gls_vans.cc

```diff
@@ -1628,51 +1645,23 @@ GLSVANSSolver<dim>::post_processing()
                    bdf_coefs[3] * p3_void_fraction_values[q]) *
                   fe_values_void_fraction.JxW(q);
 
-              // Calculation of fluid and bed volumes in bed
-              if (present_void_fraction_values[q] < 1.0)
-                {
-                  fluid_volume += present_void_fraction_values[q] *
-                                  fe_values_void_fraction.JxW(q);
-
-                  bed_volume += fe_values_void_fraction.JxW(q);
-                }
-
-              mass_source += local_mass_source;
+              continuity += local_mass_source;
             }
 
-          max_local_mass_source =
-            std::max(max_local_mass_source, abs(local_mass_source));
+          max_local_continuity =
```

Comment:
I think I prefer "mass source".

Reply:
Yeah but that's problematic, because you could have a problem where you DO have a mass source. Like for example dissolving particles and etc.. So I would really rather it be the continuity equation